CHANGELOG.md

```diff
@@ -32,6 +32,7 @@
 * The {ref}`nk.hilbert.random.flip_state` method used by `MetropolisLocal` now throws an error when called on a {ref}`nk.hilbert.ContinuousHilbert` hilbert space instead of entering an endless loop. [#1014](https://github.com/netket/netket/pull/1014)
 * Fixed bug in conversion to qutip for `MCMixedState`, where the resulting shape (hilbert space size) was wrong. [#1020](https://github.com/netket/netket/pull/1020)
 * Setting `MCState.sampler` now recomputes `MCState.chain_length` according to `MCState.n_samples` and the new `sampler.n_chains`. [#1028](https://github.com/netket/netket/pull/1028)
+* The default value of `MCState.n_discard_per_chain` is now correctly set to 1/10 of the chain length, rather than 1/10 of the total number of samples. [#1034](https://github.com/netket/netket/pull/1034)
```

Comment:
I am not sure that in general it is a good idea to set n_discard with respect to the number of samples per chain. Maybe a better default would be just some constant number, say 10...

Reply:
Anyways, the choice behind the current default value is that n_discard should not depend on how many chains you have, since thermalization time is a property of the MCMC sampler and cannot be parallelized